src/riak_test_escript.erl

```diff
@@ -325,14 +325,12 @@ run_test(Test, TestType, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
                 {ok, Base} ->
                     %% Now push up the artifacts, starting with the test log
                     giddyup:post_artifact(Base, {"riak_test.log", L}),
-<<<<<<< HEAD
                     [ giddyup:post_artifact(Base, File) || File <- rt:get_node_logs() ],
```

Comment:
Lines 328-333 need to be removed.  This was an either/or decision and we are choosing *only* the `develop-2.2` changes